meeting-summary/2022-09-09-表格开领域问答.md

```diff
@@ -0,0 +1,208 @@
+# 表格开领域问答
+
+- 分享人：陈思芹
+- 记录人：陈思芹
+- 关键词：Open-domain QA, Table Retrieval
+- 分享PPT: [2022-09-09-表格开领域问答.pdf](https://github.com/Sunlly0/Weekly-Group-Meeting-Paper-List/blob/main/meeting-summary/slides/2022-09-09-表格开领域问答.pdf) 
+
+# 分享内容
+
+## 1. 背景
+
+### 任务定义
+
+Open-domain QA 即开领域问答，往往只给定了一个大型的数据源，比如百万级的文档或者大型知识库，通过搜索出和问题相关的内容并提取答案。
+
+![Untitled](imgs/image-20220909-1.png)
+
+### 发展历史
+
+ 20 世纪 60 年代， 开始QA 系统的研究，应用在体育和学术研究领域。
+
+1999 年，TREC 会议提出了 OpenQA 任务的定义，开启了 Open-domain QA 的先河。
+
+### 框架发展
+
+由于自然语言处理能力的限制，早期的开领域问答框架是以三阶段为主的。
+
+![Untitled](imgs/image-20220909-2.png)
+
+随着深度学习技术的发展，比如 CNN 和 LSTM 的出现， OpenQA 有了很大的发展，比如基于 CNN 的分类器使得问题分析中的分类准确性有了很大的提升，同时，神经网络模型使得答案提取阶段有了突破的进展。
+
+2017年，DrQA 提出了两阶段的 Retriever-Reader  框架，成为开领域问答中的主流。
+
+![Untitled](imgs/image-20220909-3.png)
+
+新框架：Retrieval Free。比如 T5，经过训练后的 T5 可以直接跳过检索阶段，直接生成答案。 目前还不太完善，但也为开领域任务提供了一个新的思路和探索方向。
+
+![Untitled](imgs/image-20220909-4.png)
+
+### 问答任务分类
+
+![Untitled](imgs/image-20220909-5.png)
+
+## 2. 检索部分
+
+### **TF-IDF**
+
+TF-IDF 是一个加权技术，可以用来衡量一个字词对于文档的重要程度。
+
+- TF(Term Frequency)，也称为词频。即某个词在文档中出现的次数。考虑到不同文档的长度是不一样的，将实际的次数除以文档的总词数进行标准化：
+  
+    ![20b6fd80a1984b77898136eca85a2234.png](imgs/image-20220909-6.png)
+    
+- IDF(Inverse Document Frequency)，也称为逆文档频率。用于衡量一个词在语料库中的稀有程度。
+  
+    ![1c6786bd77454bd5849859253bd8080f.png](imgs/image-20220909-7.png)
+    
+
+TF-IDF 的计算：将TF 和IDF 相乘
+
+![12cbdc1cb8724ff4b8f2d30f5a7cf07a.png](imgs/image-20220909-8.png)
+
+可以看到，**TF-IDF与一个词在文档中的出现次数成正比，与该词在整个语言中的出现次数成反比**。
+
+### **BM25**
+
+BM25（BM：best matching）算法是一种计算句子与文档相关性的算法。先将输入的句子分词，然后计算句子中每个词与文档的相关性，最后进行加权求和，得到句子和文档的相关性评分。
+
+计算公式如下：
+
+![Untitled](imgs/image-20220909-9.png)
+
+- D: 文档， Q：问句
+- q_i：问句中的各个分词
```

Comment:
Github has supported mathematical expressions in markdown, mathematical expression should be wrapped in `$` like that `$q_i$`, which will be render as $q_i$.

more discussion please refer to https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/discussions/31#discussioncomment-2852396